install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# This is an install script for notes
+
+function assertInstalled() {
```

Comment:
Interesting minor but bizarre problem here. If I curl to install this with:

```
curl -L https://rawgit.com/pimterry/notes/install-test/install.sh | bash
```

(literally exactly that - I've pushed this PR + a tweak to make the URL work before this is merged to the `install-test` branch here so I can curl it for testing)

then I get:

```
Checking for root...
Checking for Dependencies...
bash: line 4: assertInstalled: command not found
Downloading and Extracting Notes from Repository...
```

Note the (ineffectual) `assertInstalled` error. If I run `./install.sh` from bash within a checkout of the same code though, I get the same result, and otherwise the same output, but without that error. Any idea why?